webpack.config.js

```diff
@@ -66,7 +66,10 @@ module.exports = (env, argv) => {
             {
               loader: "url-loader",
               options: {
-                limit: 8192,
+                // Removed `limit: 8192,` because images were disappearing after
```

Comment:
I'm a lil surprised that the limit in the demo/sample was so low. Doc suggests that default is [undefined](https://www.npmjs.com/package/url-loader#limit), and I'm okay without a comment or explanation.

Hmm, the assets seem reasonably sized.

Reply:
might want to reinstate this if later someone adds larger assets, so I'm find with having the explanation here